tests/gold_tests/tls/tls_client_cert.test.py

```diff
@@ -25,8 +25,10 @@
 cafile = "{0}/signer.pem".format(Test.RunDirectory)
 cafile2 = "{0}/signer2.pem".format(Test.RunDirectory)
 # --clientverify: "" empty string because microserver does store_true for argparse, but options is a dictionary
-server = Test.MakeOriginServer("server", ssl=True, options = { "--clientCA": cafile, "--clientverify": ""}, clientcert="{0}/signed-foo.pem".format(Test.RunDirectory), clientkey="{0}/signed-foo.key".format(Test.RunDirectory))
-server2 = Test.MakeOriginServer("server2", ssl=True, options = { "--clientCA": cafile2, "--clientverify": ""}, clientcert="{0}/signed2-bar.pem".format(Test.RunDirectory), clientkey="{0}/signed-bar.key".format(Test.RunDirectory))
+server = Test.MakeOriginServer("server", ssl=True, options={"--clientCA": cafile, "--clientverify": ""}, clientcert="{0}/signed-foo.pem".format(
```

Comment:
It would be nice if we could split this line like:
```python
server = Test.MakeOriginServer("server",
                               ssl=True,
                               options={"--clientCA": cafile, "--clientverify": ""},
                               clientcert="{0}/signed-foo.pem".format(Test.RunDirectory),
                               clientkey="{0}/signed-foo.key".format(Test.RunDirectory))
```

Reply:
I agree that looks better.

I was initially writing a reply saying I didn't think that I'd be able to get autopep8 to format like this (it's not as configurable as clang-format is). But playing around with things a little it looks like I can get that to work if I pass in the `--aggressive --aggressive` argument to autopep8.

My latest commit uses that. I think it's better.

Thanks for the feedback.